Automattic/newspack-plugin
# Feat/1393 update plugin list

### All Submissions:


### Changes proposed in this Pull Request:

Update plugin list to remove the less-recommended plugins. Add Gravity Forms. Make escaping and translation more consistent. Closes #1393 .

### How to test the changes in this Pull Request:

1. go to the plugins screen in wp-admin
2. verify that the plugins list is updated, plugins are installable, etc

### Other information:

* [N/A] Have you written new tests for your changes, as applicable?


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201069149097200
